source/speech/speech.py

```diff
@@ -1073,17 +1079,15 @@ def speak(  # noqa: C901
 	@param symbolLevel: The symbol verbosity level; C{None} (default) to use the user's configuration.
 	@param priority: The speech priority.
 	"""
+	pre_filter_speechSequence.notify(sequence=speechSequence)
 	speechSequence = filter_speechSequence.apply(speechSequence)
 	logBadSequenceTypes(speechSequence)
 	# in case priority was explicitly passed in as None, set to default.
 	priority: Spri = Spri.NORMAL if priority is None else priority
 
 	if not speechSequence:  # Pointless - nothing to speak
 		return
-	import speechViewer
-
-	if speechViewer.isActive:
-		speechViewer.appendSpeechSequence(speechSequence)
+	post_filter_speechSequence.notify(sequence=speechSequence)
 	pre_speech.notify(speechSequence=speechSequence, symbolLevel=symbolLevel, priority=priority)
```

Comment:
since these are called at the same point in time, aren't they semantically the same?
I don't think we need `post_filter_speechSequence` any more, `pre_speech` is the same concept.